demos/adc/README.md

```diff
@@ -1 +1,3 @@
-Reads ADC channels ADC10, ADC11, ADC12.
+This demo reads ADC channels ADC10, ADC11, ADC12. The ADC conversion group is
+[ADC10, ADC11, ADC12] and is read at approximately 1 kHz. The ADC values are
+transmitted over serial in ASCII in units of bits.
```

Comment:
The ADC driver we use is 12-bit and converts the voltage range 0V to 3.3V to some digital value between 0 and 4095. ChibiOS calls this a `adcsample` so I'm not sure what the best word to use is.